backend/test_observer/controllers/artefacts/models.py

```diff
@@ -46,6 +48,8 @@ class ArtefactDTO(BaseModel):
     stage: str = Field(validation_alias=AliasPath("stage", "name"))
     status: ArtefactStatus
     assignee: UserDTO | None
+    due_date: date | None
+    bug_link: str
```

Comment:
For other links we use `Annotated[str, HttpUrl]`, should we maybe do same here?

Reply:
Those are requests, while this is a response. While we want to validate data going into our application to make sure it's clean, we probably shouldn't validate data going out of the application. Otherwise if say an artefact for some reason had a malformed link, then it would just never be returned as that endpoint would fail

Reply:
Ah good point, please ignore it.